pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/default.nix

```diff
@@ -1,38 +1,42 @@
 { mkDerivation, lib, fetchFromGitHub, pkg-config, python3, cmake, ninja
 , qtbase, qtimageformats, libdbusmenu, hunspell, xdg-utils, ffmpeg_3, openalSoft
-, lzma, lz4, xxHash, zlib, minizip, openssl, libtgvoip, microsoft_gsl, tl-expected
```

Comment:
Doesn't seem to be used with `-DDESKTOP_APP_USE_PACKAGED_GSL=OFF` (BTW, it doesn't build without this flag).